kolibri/plugins/facility/assets/src/views/ManageClassPage/index.vue

```diff
@@ -145,24 +145,28 @@
             dataType: 'string',
             minWidth: '150px',
             width: '20%',
+            columnId: this.coreString('classNameLabel'),
```

Comment:
Why are you using the translated string here for the `columnId`? Is it going to be user-facing or just internal to KTable?

Reply:
It's supposed to be internal.

Reply:
I think we can omit using translated strings here then, since it is unnecessary for internal usage.